src/v/wasm/tests/wasm_fixture.cc

```diff
@@ -149,6 +149,9 @@ void WasmTestFixture::SetUp() {
           .debug_host_stack_usage = false,
 #endif
         },
+        .cpu = {
+          .per_invocation_timeout = 3ms,
```

Comment:
Is `3ms` intentional here? (rather than the `3s` default for the the config)

Reply:
Ha, good catch! Thanks, fixed.